bindings/python/README.md

```diff
@@ -76,74 +76,74 @@ returns probability for each frame) and a lexicon (mapping between words and the
-    # token_dict.add_entry("1")
-    # token_dict.add_entry("2")
+tokens_dict = Dictionary("path/tokens.txt")
+# for ASG add used repetition symbols, for example
+# token_dict.add_entry("1")
+# token_dict.add_entry("2")
 
-    lexicon = load_words("path/lexicon.txt") # returns LexiconMap
-    word_dict = create_word_dict(lexicon) # returns Dictionary
+lexicon = load_words("path/lexicon.txt") # returns LexiconMap
+word_dict = create_word_dict(lexicon) # returns Dictionary
 ```
 
 To create a KenLM language model, use:
 
 ```python
-    from flashlight.lib.text.decoder import KenLM
-    lm = KenLM("path/lm.arpa", word_dict) # or "path/lm.bin"
+from flashlight.lib.text.decoder import KenLM
+lm = KenLM("path/lm.arpa", word_dict) # or "path/lm.bin"
 ```
 
 Get the unknown and silence token indices from the token and word dictionaries to pass to the decoder:
 
 ```python
-    sil_idx = token_dict.get_index("|")
-    unk_idx = word_dict.get_index("<unk>")
+sil_idx = token_dict.get_index("|")
+unk_idx = word_dict.get_index("<unk>")
 ```
 
 Now, define the lexicon `Trie` to restrict the beam search decoder search:
 
 ```python
-    from flashlight.lib.text.decoder import Trie, SmearingMode
-    from flashlight.lib.text.dictionary import pack_replabels
+from flashlight.lib.text.decoder import Trie, SmearingMode
+from flashlight.lib.text.dictionary import pack_replabels
 
-    trie = Trie(token_dict.index_size(), sil_idx)
-    start_state = lm.start(False)
+trie = Trie(token_dict.index_size(), sil_idx)
+start_state = lm.start(False)
 
-    def tkn_to_idx(spelling: list, token_dict : Dictionary, maxReps : int = 0):
-        result = []
-        for token in spelling:
-            result.append(token_dict.get_index(token))
-        return pack_replabels(result, token_dict, maxReps)
+def tkn_to_idx(spelling: list, token_dict : Dictionary, maxReps : int = 0):
+    result = []
+    for token in spelling:
+        result.append(token_dict.get_index(token))
+    return pack_replabels(result, token_dict, maxReps)
 
 
-    for word, spellings in lexicon.items():
-        usr_idx = word_dict.get_index(word)
-        _, score = lm.score(start_state, usr_idx)
-        for spelling in spellings:
-            # convert spelling string into vector of indices
-            spelling_idxs = tkn_to_idx(spelling, token_dict, 1)
-            trie.insert(spelling_idxs, usr_idx, score)
+for word, spellings in lexicon.items():
+    usr_idx = word_dict.get_index(word)
+    _, score = lm.score(start_state, usr_idx)
+    for spelling in spellings:
+        # convert spelling string into vector of indices
+        spelling_idxs = tkn_to_idx(spelling, token_dict, 1)
+        trie.insert(spelling_idxs, usr_idx, score)
 
-        trie.smear(SmearingMode.MAX) # propagate word score to each spelling node to have some lm proxy score in each node.
+trie.smear(SmearingMode.MAX) # propagate word score to each spelling node to have some lm proxy score in each node.
```

Comment:
This one is unindented more than the preceding for-loop. My understanding is that `smear` needs to be called only once, so I believe this is the intended change.

Reply:
You're totally right, thanks for the catch